components/automate-chef-io/README.md

```diff
@@ -10,50 +10,56 @@ A new static website for Automate is based on [Hugo](https://gohugo.io), the Go-
 
 1. [Install Hugo](https://gohugo.io/getting-started/installing/). If you're using homebrew:
 
-    ```
+    ```shell
     brew install hugo
     ```
 
 1. [Install go-swagger](https://goswagger.io/install.html). The currently supported version is 0.20.1. If you're using homebrew:
 
-    ```
-    brew tap go-swagger/go-swagger
-    brew install go-swagger
+    ```shell
+    brew tap go-swagger/go-swagger && brew install go-swagger
     ```
 
     If everything is up to date you should see output like this when running `swagger version`:
 
-    ```
-    $ swagger version
-    version: v0.21.0
-    commit: 7c0fc3ee340f9d99d85573bc54a57e303a639692
+    ```shell
+    $ Swagger version
+    version: v0.20.1
+    commit: 9004771f77bedabb792a48dc846ba7fc800398a0
```

Comment:
0.21.0 is the latest version so this change should probably be reverted. I'm not sure how important it is that the instructions be kept up to date with the latest latest version anymore though, so maybe we should just remove this?